struct.go

```diff
@@ -326,8 +341,34 @@ func (s *Section) mapTo(val reflect.Value, isStrict bool) error {
 	return nil
 }
 
-// MapTo maps section to given struct.
-func (s *Section) MapTo(v interface{}) error {
+// maps all sections with the same name and returns the new value.
+func (s *Section) mapAllTo(secName string, val reflect.Value, isStrict bool) (reflect.Value, error) {
```

Comment:
Please use defensive programming here, return early to save some indentations.

Reply:
I am not sure what exactly you mean. Where can I return more early?